code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -516,3 +516,79 @@
 		human_user.adjustStaminaLoss(-10, FALSE)
 		human_user.adjustToxLoss(-10, FALSE)
 		human_user.adjustOxyLoss(-10)
+
+/obj/effect/proc_holder/spell/pointed/manse_link
+	name = "Mansus Link"
+	desc = "Piercing through reality, connecting minds. This spell allows you to add people to a mansus net, allowing them to communicate with eachother"
+	school = "transmutation"
+	charge_max = 300
+	clothes_req = FALSE
+	invocation = "PI'RC' TH' M'ND"
+	invocation_type = "whisper"
+	range = 10
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "mansus_link"
+	action_background_icon_state = "bg_ecult"
+
+/obj/effect/proc_holder/spell/pointed/manse_link/can_target(atom/target, mob/user, silent)
+	if(!isliving(target))
+		return FALSE
+	return TRUE
+
+/obj/effect/proc_holder/spell/pointed/manse_link/cast(list/targets, mob/user)
+	var/mob/living/simple_animal/hostile/eldritch/raw_prophet/originator = user
+
+	var/mob/living/target = targets[1]
+
+	to_chat(originator, "<span class='notice'>You begin linking [target]'s mind to yours...</span>")
+	to_chat(target, "<span class='warning'>You feel your mind being pulled... connected... intertwined with the very fabric of reality...</span>")
+	if(!do_after(originator, 6 SECONDS, target))
+		return
+	if(!originator.link_mob(target))
+		to_chat(originator, "<span class='warning'>You can't seem to link [target]'s mind...</span>")
+		to_chat(target, "<span class='warning'>The foreign presence leaves your mind.</span>")
+		return
+	to_chat(originator, "<span class='notice'>You connect [target]'s mind to your mansus link!</span>")
+
+
+/datum/action/innate/mansus_speech
+	name = "Mansus Link"
+	desc = "Send a psychic message to everyone connected to your mansus link."
+	button_icon_state = "link_speech"
+	icon_icon = 'icons/mob/actions/actions_slime.dmi'
+	background_icon_state = "bg_ecult"
+	var/mob/living/simple_animal/hostile/eldritch/raw_prophet/originator
+
+/datum/action/innate/mansus_speech/New(_originator)
+	. = ..()
+	originator = _originator
+
+/datum/action/innate/mansus_speech/Activate()
+	var/mob/living/living_owner = owner
+	if(!originator || !(living_owner in originator.linked_mobs))
```

Comment:
Can these checks be triggered? Doesn't the action get removed in deletion of the `originator` or removal of `living_owner` from `linked_mobs`?

Reply:
This is just in case something goes terribly wrong, so we dont runtime every time someone tries to activate it.

Reply:
I don't see you unlinking or doing anything with the mob or action lists if the prophet dies or gets deleted.

Reply:
If something goes terribly wrong we do want things to runtime and scream in error. There's nothing worse than silent errors for debugging.

Reply:
aight

Reply:
done

Reply:
```suggestion
	if(!originator?.linked_mobs[living_owner])
```